usb/backend/libusb1.py

```diff
@@ -731,6 +731,13 @@ def get_device_descriptor(self, dev):
         except AttributeError:
             dev_desc.port_numbers = None
 
+        try:
+            dev_desc.speed = self.lib.libusb_get_device_speed(dev.devid)
```

Comment:
It is a convention supply the function prototype to avoid argument type mistakes.